lib/job_base.cu

```diff
@@ -0,0 +1,71 @@
+#include <include/job_base.h>
+
+// constructor and initialization
+JobBase::JobBase(){
+    partition = 1 / size_of_process_time;
```

Comment:
`(1 / size_of_process_time)` is an integer number since both operands are unsigned int type. Next version, you have to use type casting to change the type. Thanks !!
`partition = 1.0 / (double)size_of_process_time;`